src/domains/shared-app/api/session-wallet/v1/_base_policy.xml.tpl

```diff
@@ -173,30 +173,57 @@
             </choose>
             <set-variable name="userAuthBody" value="@(((IResponse)context.Variables["user-auth-body"]).Body.As<JObject>())" />
             <!-- Get User IO : END-->
+
+            <!-- user email tokenization with PDV START -->
+            <send-request ignore-error="true" timeout="10" response-variable-name="pdv-email-token" mode="new">
+              <set-url>${pdv_api_base_path}/tokens</set-url>
+              <set-method>PUT</set-method>
+              <set-header name="x-api-key" exists-action="override">
+                  <value>{{ecommerce-personal-data-vault-api-key}}</value>
```

Comment:
what about use here `wallet-session-personal-data-vault-api-key` so that this claim can be de-tokenized with the same key used for userId? Maybe this policy have to tokenize all data with the same key, what do you think?

Reply:
Right now on eCommerce we have a separate key for emails, if we revisit this decision we can unify the two keys